package.json

```diff
@@ -2,6 +2,10 @@
   "name": "dojo-core",
   "version": "2.0.0-pre",
   "description": "Basic utilites for common TypeScript development",
+  "engines": {
+    "node": "^6.5.0",
```

Comment:
do you think we should just put 6.0.0 (which is the minimum version)? I am running `6.3.0`